src/platform/error_abi.cpp

```diff
@@ -88,7 +102,8 @@ namespace xlang::impl
 
         void GetProjectionIdentifier(xlang_string* projection_identifier) noexcept override
         {
-            *projection_identifier = m_projection_identifier;
+            *projection_identifier = nullptr;
```

Comment:
I think you can/should safely assume that copy_to_abi will clear the out param. If that's not the case then we should make it so.

Reply:
agreed. Since it has no failure mode, it must always set its out parameter to a stable value.

Reply:
copy_to_abi seems to taken in a reference and asserts that the passed reference == nullptr.